### PR TITLE
Add support for main dc schema.mcf.

### PR DESCRIPTION
**Pull request by keyurva**
* Currently creates `schema.mcf` only for SVs and SVGs.

### CODE REVIEW

File: simple/sample/main_dc_output/schema.mcf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+typeOf: StatisticalVariable
+name: "Good var1 name"
+description: "Good var1 description"
+memberOf: custom/g/group_2
+
+Node: dcid:var2
+typeOf: StatisticalVariable
+name: "Good var2 name"
+memberOf: custom/g/group_3
+
```
Comment (pradh) [inline simple/sample/main_dc_output/schema.mcf:31]: This will fail validation.  SV needs properties like populationType, measuredProperty, statType?

Comment (keyurva) [reply simple/sample/main_dc_output/schema.mcf:35]: Added populationType and measuredProperty. PLMK if they look correct.

As for statType - should it be something users need to specify in a config? And what should be the default?

Comment (pradh) [reply simple/sample/main_dc_output/schema.mcf:35]: https://github.com/datacommonsorg/data/blob/master/docs/representing_statistics.md#statisticalvariable 

`measuredValue` is the default

File: simple/stats/data.py [hunk 2]
```diff
@@ -31,6 +31,8 @@
 _PREDICATE_SUB_CLASS_OF = "subClassOf"
 _PREDICATE_OBSERVATION_DATE = "observationDate"
 _PREDICATE_LOCATION = "location"
+_PREDICATE_POPULATION_TYPE = "populationType"
+_PREDICATE_MEASURED_PROPERTY = "measuredProperty"
```
Comment (pradh) [inline simple/stats/data.py:35]: I we should support statType too.  And everything else in https://github.com/datacommonsorg/data/blob/master/docs/representing_statistics.md#statisticalvariable ?

**Review by pradh — APPROVED** [review]
If you want to add these in follow up PR so this can go in, given current RTT, approving this.

File: simple/sample/main_dc_output/schema.mcf (hunk 1 above)
Comment (keyurva) [reply simple/sample/main_dc_output/schema.mcf:35]: Perfect. Filed b/322909728

File: simple/stats/data.py (hunk 2 above)
Comment (keyurva) [reply simple/stats/data.py:35]: SG. b/322909728

**Comment by keyurva** [comment]
> If you want to add these in follow up PR so this can go in, given current RTT, approving this.

Thanks. Yes, will add in a follow up PR.

**Pull request merged**